### PR TITLE
[aptos-cli] use aptos-transaction-builder instead of hand written functions

### PR DESCRIPTION
**Pull request by davidiw**
hand written functions provide no guarantees and we can see that
increase lockup was actually broken.

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2529)

### CODE REVIEW

File: crates/aptos/src/node/mod.rs [hunk 1]
```diff
@@ -184,26 +158,8 @@ impl CliCommand<Transaction> for IncreaseLockup {
     }
 
     async fn execute(mut self) -> CliTypedResult<Transaction> {
-        if self.lockup_duration.is_zero() {
-            return Err(CliError::CommandArgumentError(
-                "Must provide a non-zero lockup duration".to_string(),
-            ));
-        }
```
Comment (gregnazario) [inline crates/aptos/src/node/mod.rs:191]: What happened to the lockup duration?

File: crates/aptos/src/node/mod.rs [hunk 2]
```diff
@@ -381,20 +337,14 @@ impl CliCommand<Transaction> for RegisterValidatorCandidate {
                 }
             };
 
+        // Double BCS encode, so that we can hide the original type
```
Comment (gregnazario) [inline crates/aptos/src/node/mod.rs:340]: Can you put this back down next to the actual double encoding?

**Review by gregnazario — APPROVED** [review]

File: crates/aptos/src/node/mod.rs (hunk 1 above)
Comment (davidiw) [reply crates/aptos/src/node/mod.rs:191]: I'm just the messenger...

But the new model just increases the lockup to the current maximum.

File: crates/aptos/src/node/mod.rs (hunk 2 above)
Comment (davidiw) [reply crates/aptos/src/node/mod.rs:340]: oh good call

**Pull request merged**